rollup.config.js

```diff
@@ -0,0 +1,117 @@
+import commonjs from '@rollup/plugin-commonjs';
+import resolve from '@rollup/plugin-node-resolve';
+import url from '@rollup/plugin-url';
+import babel from 'rollup-plugin-babel';
+import peer from 'rollup-plugin-peer-deps-external';
+import postcss from 'rollup-plugin-postcss';
+import del from 'rollup-plugin-delete';
+import { terser } from 'rollup-plugin-terser';
+
+import pkg from './package.json';
+
+const umdOutputOptions = {
+  name: 'InfiniteCalendar',
+  format: 'umd',
+  exports: 'named',
+  globals: {
+    react: 'React',
+    'react-dom': 'reactDom',
+    'react-transition-group': 'reactTransitionGroup',
+  },
+};
+
+const plugins = [
+  url(),
+  babel({
+    presets: [['react-app', { absoluteRuntime: false, useESModules: false }]],
+    plugins: [
+      '@babel/plugin-proposal-object-rest-spread',
+      '@babel/plugin-proposal-optional-chaining',
+      '@babel/plugin-syntax-dynamic-import',
+      '@babel/plugin-proposal-class-properties',
+      'transform-react-remove-prop-types',
+    ],
+    exclude: 'node_modules/**',
+    runtimeHelpers: true,
+  }),
+  resolve(),
+];
+
+export default [
+  {
+    input: 'src/index.js',
+    output: [
+      {
+        dir: pkg.main,
+        format: 'cjs',
+        exports: 'named',
+      },
+      {
+        dir: pkg.module,
+        format: 'es',
+        exports: 'named',
+      },
+    ],
+    external: id => id.startsWith('@babel/runtime/helpers'),
+    preserveModules: true,
+    plugins: [
+      ...plugins,
+      commonjs(),
+      peer({
```

Comment:
This won't include the peerDependencies right?

Reply:
I'm actually not sure even if this include the dependencies; because we'll duplicate date-fns helpers used on the aa-react side too.

Reply:
@SBoudrias This is actually to config peer dependencies as externals, and with `includeDependencies: true` we treat all the dependencies as external, so none of them will be bundled